source/Reloaded.Mod.Launcher.Lib/Misc/Constants.cs

```diff
@@ -9,6 +9,7 @@ public static class Constants
     public const string ParameterKill = "--kill";
     public const string ParameterLaunch = "--launch";
     public const string ParameterArguments = "--arguments";
+    public const string WorkingDirectory = "--working-directory";
```

Comment:
This should probably be renamed to match the rest of the CLI parameters.

Reply:
whoops